lib/rules/no-multi-spaces.js

```diff
@@ -12,7 +14,17 @@
 module.exports = function(context) {
 
     // the index of the last comment that was checked
-    var lastCommentIndex = 0;
+    var exceptions = { "Property": true },
+        exceptionCount = 1,
+        options = context.options[0],
+        lastCommentIndex = 0;
+
+    if (options && options.exceptions) {
+        Object.keys(options.exceptions).forEach(function (key) {
+            exceptions[key] = options.exceptions[key];
+        });
+        exceptionCount = Object.keys(exceptions).length;
```

Comment:
Hmm... is the page cached? This portion is showing as an outdated diff for me. If I go to the Files changed tab this section is an if/else, setting in the consequent and deleting in the alternate.